src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-filed">
+              Usarname:
```

Comment:
```suggestion
      Usarname:
```

Reply:
<img width="900" alt="image" src="https://user-images.githubusercontent.com/102411130/222072669-5a1aa0b4-96ad-410b-a0b9-5b6d2d330a23.png">
add label to all inputs